pkg/panels/kubernetes/cluster.go

```diff
@@ -0,0 +1,484 @@
+			),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"sum(namespace_memory:kube_pod_container_resource_requests:sum{cluster=\"$cluster\"}) by (namespace)",
+					labelMatchers,
+				),
+				dashboards.AddQueryDataSource(datasourceName),
+			),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"sum(container_memory_rss{"+GetCAdvisorMatcher()+", cluster=\"$cluster\", container!=\"\"}) by (namespace) / sum(namespace_memory:kube_pod_container_resource_requests:sum{cluster=\"$cluster\"}) by (namespace)",
+					labelMatchers,
+				),
+				dashboards.AddQueryDataSource(datasourceName),
+			),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"sum(namespace_memory:kube_pod_container_resource_limits:sum{cluster=\"$cluster\"}) by (namespace)",
+					labelMatchers,
+				),
+				dashboards.AddQueryDataSource(datasourceName),
+			),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"sum(container_memory_rss{"+GetCAdvisorMatcher()+", cluster=\"$cluster\", container!=\"\"}) by (namespace) / sum(namespace_memory:kube_pod_container_resource_limits:sum{cluster=\"$cluster\"}) by (namespace)",
+					labelMatchers,
+				),
+				dashboards.AddQueryDataSource(datasourceName),
+			),
+		),
+	)
+}
+
+func ClusterCurrentNetworkUsage(datasourceName string, labelMatchers ...promql.LabelMatcher) panelgroup.Option {
+	return panelgroup.AddPanel("Current Network Usage",
+		panel.Description("Shows the current network usage of the cluster by namespace."),
+		tablePanel.Table(
+			tablePanel.Transform([]commonSdk.Transform{
+				{
+					Kind: commonSdk.MergeSeriesKind,
+					Spec: commonSdk.MergeSeriesSpec{
+						Disabled: false,
+					},
+				},
+			}),
+			tablePanel.WithColumnSettings([]tablePanel.ColumnSettings{
+				{
+					Name:   "namespace",
+					Header: "Namespace",
+					Align:  tablePanel.LeftAlign,
+				},
+				{
+					Name:   "value #1",
+					Header: "Current Receive Bandwidth",
+					Align:  tablePanel.RightAlign,
+				},
+				{
+					Name:   "value #2",
+					Header: "Current Transmit Bandwidth",
+					Align:  tablePanel.RightAlign,
+				},
+				{
+					Name:   "value #3",
+					Header: "Rate of Received Packets",
+					Align:  tablePanel.RightAlign,
+				},
+				{
+					Name:   "value #4",
+					Header: "Rate of Transmitted Packets",
+					Align:  tablePanel.RightAlign,
+				},
+				{
+					Name:   "value #5",
+					Header: "Rate of Received Packets Dropped",
+					Align:  tablePanel.RightAlign,
+				},
+				{
+					Name:   "value #6",
+					Header: "Rate of Transmitted Packets Dropped",
+					Align:  tablePanel.RightAlign,
+				},
+				{
+					Name: "timestamp",
+					Hide: true,
+				},
+			}),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"sum(rate(container_network_receive_bytes_total{"+GetCAdvisorMatcher()+", cluster=\"$cluster\", namespace=~\".+\"}[5m])) by (namespace)",
```

Comment:
Question(nit): Does it make sense to leverage the built_in prometheus variable `$__rate_interval` instead of hardcoding [5m]? I have the same question for all the other panels with interval as well. 
```
					"sum(rate(container_network_receive_bytes_total{"+GetCAdvisorMatcher()+", cluster=\"$cluster\", namespace=~\".+\"}[$__rate_interval])) by (namespace)",
```

Reply:
Right, so this is kinda broken currently with the promql functions here (can't pass variables to parsers). I have a plan to fix it in next PR. See: https://github.com/perses/community-dashboards/pull/52#discussion_r2077969718

Reply:
Awesome! Your fix will also help all the other existing dashboards! 🚀